src/bin/log/log.ml

```diff
@@ -0,0 +1,2 @@
+let src = Logs.Src.create "deku.cli"
+include (val Logs.src_log src : Logs.LOG)
```

Comment:
This is not what we want IMO. We want a few functions (that you have to define yourself) matching the various levels of logging, for `app` (should be used by deku_node and people using deku as a library only), `info` (should be used by us mostly, when we produce blocks or update the hash), `debug` and `error`. These functions should be used as `Logs.info "New protocol snapshot hash: %s" (BLAKE2B.to_string hash)` directly in Deku.

I'll let @oteku @d4hines decide if we want colorized output or not.

Also, move this to helpers please.

Reply:
I should have been more clear about not using include. Also, I guess I was wrong about `app` being used in the deku_node module.